store/index.js

```diff
@@ -25,4 +27,17 @@ export const actions = {
         const keynoteList = await this.$http.$get('/api/events/keynotes/')
         commit('setKeynotesData', keynoteList)
     },
+    async $verifyAttendee({ commit }, payload = { token: '' }) {
+        try {
```

Comment:
nit: 

we can register `onError` hook instead of `try...catch`
https://http.nuxtjs.org/advanced#register-hooks